src/static/css/base.css

```diff
@@ -8,11 +8,21 @@
 	url('../fonts/ArialRoundedMTBold.svg#svgFontName') format('svg'); /* Legacy iOS */
 }
 
+@font-face {
+	font-family: "Eurostile";
+	src: url("../fonts/Eurostile.ttf");
```

Comment:
It may be worth converting this .ttf to all of the other types with https://onlinefontconverter.com/. It actually looks like TTF has good browser support, but it would be a safe move to have them all. Also, the browser might optimize by using the most efficient file type. You might also be interested to see details on web font format efficiencies here: https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization